tomland.cabal

```diff
@@ -43,7 +43,7 @@ library
 
   build-depends:       base >= 4.10 && < 5
                      , hashable ^>= 1.2
-                     , megaparsec ^>= 6.5.0
+                     , megaparsec ^>= 7.0
```

Comment:
Could you specify `^>= 7.0.1` version here? I see from the `megaparsec` changelog that `7.0.1` version contains bugfix, so I would like to constraint version even more to be sure that we have the bugfix.